commands/python/command/start.py

```diff
@@ -62,9 +88,11 @@ def start(namespace, dry_run=False, with_mount=False, force_helm=False, set_over
     value_overrides = {k: v for k, v in (value.split("=") for value in set_override_values)}
     values.update(value_overrides)
 
-    if dry_run:
-        helm.dry_run(hr, pc.helm_path, cr.cluster_name, namespace, **values)
-    else:
-        helm.start(hr, pc.helm_path, cr.cluster_name, namespace, force_helm, **values)
-        sync_ingress.sync_ingress(namespace)
-        sync_dns.sync_dns(namespace)
+    for chart_path in pc.get_helm_chart_paths():
+        if os.path.basename(chart_path) in charts:
+            if dry_run:
+                helm.dry_run(hr, chart_path, cr.cluster_name, namespace, **values)
+            else:
+                helm.start(hr, chart_path, cr.cluster_name, namespace, force_helm, **values)
+                sync_ingress.sync_ingress(namespace)
+                sync_dns.sync_dns(namespace)
```

Comment:
can sync ingress and sync dns be done just once after all charts have been started? Asking because of for loop at line 91

Reply:
yeah, i think i'll move it out and only perform the sync if any `helm.start()` call succeeds.